docs/high-performance-order-storage/cli-tools.md

```diff
@@ -0,0 +1,359 @@
+---
+post_title: HPOS CLI Tools
+tags: reference
+---
+## Overview
+
+With HPOS we introduced a set of [WP-CLI commands](https://developer.woocommerce.com/docs/category/wc-cli/) living under the `wp wc hpos` and `wp wc cot` namespaces.
+
+The following table provides an overview of what each command does, while more details and examples can be found below.
+
+Keep in mind that the commands themselves have documentation and examples that can be accessed via WP-CLI's help by passing the `--help` flag.
+
+|Command|Use this command to...|
+|----|-----|
+|`wc hpos status`|Get an overview of all HPOS matters on your site.|
+|`wc cot enable`|Enable HPOS (and possibly compatibility mode).|
+|`wc cot disable`|Disable HPOS (and possibly compatibility mode).|
+|`wc cot count_unmigrated`|Get a count of all orders pending sync.|
+|`wc cot sync`|Performantly sync orders from the currently active order storage to the other.|
+|`wc cot verify_cot_data`|Verify data between datastores.|
+|`wc hpos diff`|Get an user friendly version of the differences for an order between both order storages.|
+|`wc hpos backfill`|Copy over whole orders or specific bits of order data from any order storage to the other.|
+|`wc hpos cleanup`|Remove order data from legacy tables.|
```

Comment:
This makes me wonder if we should alias the `cot` commands to `hpos` for consistency. Or, if not, we might want to add a note here to call out the fact that the namespace is different for some of these commands.